utils/post_utils.ts

```diff
@@ -426,13 +428,15 @@ export function makeCreateAriaLabelForPost(): (state: GlobalState, post: Post) =
         (state: GlobalState, post: Post) => getReactionsForPost(state, post.id),
         (state: GlobalState, post: Post) => get(state, Preferences.CATEGORY_FLAGGED_POST, post.id, null) != null,
         getEmojiMap,
-        (post, author, reactions, isFlagged, emojiMap) => {
-            return (intl: IntlShape) => createAriaLabelForPost(post, author, isFlagged, reactions ?? {}, intl, emojiMap);
+        (state: GlobalState) => getUsersByUsername(state),
```

Comment:
I just realized this probably doesn't actually improve the theoretical performance like I thought since any change to `users` causes the mentions selector to recalculate and return a new array each time anyway. Thanks for changing that though since it'll make the logic a bit cleaner by keeping it separate